.gitpod.yml

```
@@ -0,0 +1,26 @@
+image:
+  file: .gitpod.Dockerfile
+
+tasks:
+  - name: Pip installs
+    init: pip install .[tests]
```

```suggestion
    init: pip install flake8==3.8.4 .[tests]
```

Install `flake8`, for same reasons as here; https://github.com/hyperledger/indy-node/pull/1718#pullrequestreview-840941236